tests/scripts/all.sh

```diff
@@ -3503,6 +3503,59 @@ component_test_malloc_0_null () {
     tests/ssl-opt.sh -e 'proxy'
 }
 
+support_build_aesni() {
```

Comment:
Please don't create auxiliary functions in the `support_xxx` namespace that don't match a `component_xxx` function. That would get confusing if we add a component by that name (because the existing support function would be applied to the new component).